dapps/marketplace/src/pages/identity/mutations/DeployIdentity.js

```diff
@@ -26,8 +26,13 @@ class DeployIdentity extends Component {
         {upsertIdentity => (
           <>
             <button
-              className={this.props.className}
+              className={`${this.props.className} ${this.props.disabled &&
```

Comment:
This will show a class of 'undefined' unless it resolves to true... should be `${this.props.disabled ? ' disabled' : ''}`

Reply:
you are right, this was sloppy from my side. Fixes, thanks!